src/main/java/axoloti/utils/Preferences.java

```diff
@@ -75,6 +78,9 @@
     @Element(required = false)
     String themePath;
 
+    @Element(required = false)
+    String Platform;
```

Comment:
Good point. I suspect that I probably added themePath quickly without checking the entire file or maybe even ran into the serialization change issue and avoided doing it. It's just more jarring to my eye explicitly adding a field with an initial capital. The serialization point is a painful one for sure; clearly that'd need to be resolved to change the style which is probably more pain that it's worth.